backend/Controllers/auth/verifyUser.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
+const verifyAccessToken = (req, res, next) => {
```

Comment:
Don't use arrow functions, JSDocs are much cleaner for normal functions and they are easily supported by IDEs such as VSCode